tox.ini

```diff
@@ -13,10 +13,12 @@ setenv =
    OS_STDOUT_CAPTURE=1
    OS_STDERR_CAPTURE=1
    OS_TEST_TIMEOUT=60
-deps = -c{env:UPPER_CONSTRAINTS_FILE:https://releases.openstack.org/constraints/upper/ussuri}
+deps = -c{env:UPPER_CONSTRAINTS_FILE:https://releases.openstack.org/constraints/upper/yoga}
```

Comment:
Wondering why this uses the upstream contraints instead of the sapcc ones.

Reply:
Good question. I think because that was part of the cookiecutter template I used when creating this.